pkg/io/broadcast.go

```diff
@@ -0,0 +1,136 @@
+package io
+
+import (
+	"fmt"
+	"sync/atomic"
+	"time"
+)
+
+const (
+	maskReading         = 1 << 63
+	broadcasterRingSize = 32
+	// TODO: If the data source has fps greater than 30, they'll see some
+	// 			 fps fluctuation. But, 30 fps should be enough for general cases.
+	broadcasterRingPollDuration = time.Millisecond * 33
```

Comment:
I think it should be a parameter of the broadcaster instance.
Minimal frame size of OPUS codec is 2.5ms. (usually 20ms or 40ms?)

Reply:
Oh right. So, I guess, default should be 60fps then? I don't want the default to go lower than that because it'll cause high CPU usage.

Reply:
Hmm, I think it's better to be set as an argument of NewBroadcaster.

Reply:
Updated. The ring size and poll duration are now configurable.